app/assets/stylesheets/content/_timelines.scss

```diff
@@ -125,7 +125,7 @@ table.timelines-dates {
     }
   }
   &.timelines-pt td {
-    &.timelines-pt-reorder, &.timelines-pt-allows_association, &.timelines-pt-actions {
+    &.timelines-pt-reorder, &.timelines-pt-allows_association {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `timelines-pt-allows_association` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line